internal/messages/errormessages.go

```diff
@@ -39,6 +39,7 @@ const (
 	GroupName          = "groupName"
 	Name               = "name"
 	GroupIDs           = "groupIds"
```

Comment:
it's interesting that we have now singular and plural forms for several nouns

Reply:
GropuNames removed